README.md

```diff
@@ -40,87 +40,51 @@ for information regarding the configuration of repositories in ArcLight.
 
 ### Indexing documents into Pulfalight
 
-Documents are indexed from the EADs which are stored in a subversion
-repository.
-
-#### Retrieving the EAD-XML Documents
-Documents are available from Princeton University Library staff, and should be
-populated into the `eads/pulfa` directory, yielding a structure similar to the
-following:
-
-```bash
-% ll eads/pulfa
-[...] cotsen
-[...] eng
-[...] ga
-[...] lae
-[...] mss
-[...] mudd
-[...] rarebooks
-```
-
-##### Retrieving the Documents from SVN
+#### Configuring ASpace
 
-One must ensure that SVN is installed locally:
+Create a .env file in the root of the project with the following information
 
-*In a macOS Environment:*
 ```
-brew install svn
+ASPACE_USER=yourusername
+ASPACE_PASSWORD=yourpassword
 ```
 
-###### Using `lastpass-cli` for authentication [LastPass](https://lastpass.com)
+#### Index "Interesting" EADs
+A sub-section of all our collections have been identified and can be queued up
+  for ingest via:
 
-*In a macOS Environment:*
-```
-brew install lastpass-cli
-```
+  `start redis-server`
+  `bundle exec rake pulfalight:aspace:index_test_eads`
 
```

Comment:
@tpendragon can you rephrase `start redis-server` to: 
Start redis: `redis-server` or just `redis-server`
Also can you make sure the two commands appear in separate lines?

Reply:
Updated! One more look?